### PR TITLE
Support for launching publisher login flows from Google mobile apps.

### PR DESCRIPTION
**Pull request by kprevas**

### CODE REVIEW

File: src/api/subscriptions.js [hunk 1]
```diff
@@ -271,6 +272,40 @@ export class Subscriptions {
...
   getPropensityModule() {}
 
+  /**
+   * Returns true if the Google app login flow is currently
+   * active, i.e., if googleAppLoginFlowComplete should be
+   * called when the user logs in.
+   *
+   * @return {boolean}
+   */
+  isGoogleAppLoginFlowActive() {}
```
Comment (jpettitt) [inline src/api/subscriptions.js:282]: Nit: active or available?

File: src/runtime/app-login-manager.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+ */
+
+import {tryParseJson} from "../utils/json";
+import {UserState} from '../model/user-state';
+
+const LOGIN_REDIRECT_PARAM = 'googleLoginRedirect';
+
+/**
+ */
+export class AppLoginManager {
```
Comment (jpettitt) [inline src/runtime/app-login-manager.js:24]: Can we add some logging to this to help with debugging?

File: src/api/subscriptions.js (hunk 1 above)
Comment (kprevas) [reply src/api/subscriptions.js:282]: "Active", in that this is the login page running in a CCT that was launched by the app login flow.

File: src/runtime/app-login-manager.js (hunk 2 above)
Comment (kprevas) [reply src/runtime/app-login-manager.js:25]: Done.

**Comment by ChrisAntaki** [comment]
Thanks for your formative work on the GSI flows in SwG, @kprevas !

**Pull request closed without merging**